Assets/Scripts/BehaviorTree/BlackBoard.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+[System.Serializable]
+public class BlackBoard
+{
+    public Vector3 moveToPosition;
```

Comment:
attribute in private

Reply:
Besoin car sinon plus afficher dans l'UI